pacman-nodejs-app/services/pacman-service.yaml

```diff
@@ -8,7 +8,7 @@ spec:
   type: LoadBalancer
   ports:
     - port: 80
-      targetPort: 80
+      targetPort: 8080
```

Comment:
OpenShift prevents the container from listening on port 80 without granted additional permissions, switching to 8080 avoids this issue.